streamwebs_frontend/streamwebs/models.py

```diff
@@ -64,14 +65,17 @@ def validate_UserProfile_birthdate(birthdate):
 
 
 class UserProfile(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    user = models.OneToOneField(User, on_delete=models.CASCADE,
+                                verbose_name=_('User'))
     school = models.CharField(
         max_length=255,
         choices=settings.SCHOOL_CHOICES,
         default='',
-        validators=[validate_UserProfile_school]
+        validators=[validate_UserProfile_school],
+        verbose_name=_('School')
```

Comment:
Something to keep in mind is the[ fourth bullet point of this doc](https://docs.djangoproject.com/en/1.9/topics/forms/modelforms/#field-types). Django does capitalization for you, but only the very first letter, so doing it explicitly yourself is also okay.